buildmimic/bigquery/README.md

```diff
@@ -0,0 +1,221 @@
+# Installing MIMIC3 v1.4 to BigQuery
+
+Following are the steps to create the MIMIC-III dataset on BigQuery and load the source files (.csv.gz) downloaded from Physionet.
+
+**IMPORTANT**: Only users with approved Physionet Data Use Agreement (DUA) should be given access to the MIMIC dataset via BigQuery or Cloud Storage. If you don't have access to MIMIC, follow the instructions [here](https://mimic.physionet.org/gettingstarted/access/) to request access.
+
+---
+## STEP 1: Download the MIMIC-III Source files
+
+> NOTE: According to the BigQuery documentation (Last updated March 1, 2018.), "BigQuery can load uncompressed files significantly faster than compressed files because uncompressed files can be read in parallel. Because uncompressed files are larger, using them can lead to bandwidth limitations and higher Google Cloud Storage costs for data staged in Google Cloud Storage prior to being loaded into BigQuery", and "Currently, there is no charge for loading data into BigQuery".
```

Comment:
I would lean towards uploading the files gzipped because "bq load" on compressed files is still quite fast in practice (a couple minutes) and uploading the uncompressed files can take much more time depending on the network. I switch to uncompressed files only when over the 4GiB hard limit (chartevents is just barely under). Was this a problem in your experience?